Data/Binary/Serialise/CBOR/Class.hs

```diff
@@ -193,6 +194,14 @@ instance Serialise Double where
     encode = encodeDouble
     decode = decodeDouble
 
+#if MIN_VERSION_base(4,7,0)
+-- | Values are serialised in units of least precision represented as
+--   @Integer@.
+instance HasResolution e => Serialise (Fixed e) where
+    encode (MkFixed i) = encode i
+    decode = MkFixed <$> decode
```

Comment:
Encoding Fixed as Ratio is not ideal either. It's not space efficient: we'll have to encode denominator which is always multiplier of 10^n.  And we'll have to decide what to with rationals which couldn't be represented as Fixed (1 % 3 for example). Adding constructor tag is probably better idea

And as for 7.6 I thought that its support is likely to be dropped before library release so why bother.